mlir/utils/performance/perfRunner.py

```diff
@@ -1214,6 +1224,9 @@ def main(args=None):
         configs = getConvConfigurations(paths.configuration_file_path)
     elif opType == Operation.GEMM:
         configs = getGemmConfigurations(paths.configuration_file_path)
+        # Make MLIR to be compatible with CK by forcing int8xint8->int8
```

Comment:
Will this do the right thing? Is `CKGemmConfig` the class that's being used to run the rocMLIR comparisons?

Reply:
CKGemmConfig is used if perfRunner.py is invoked with '--external-gemm-library CK'

Reply:
... right, these ones have `benchmarkExternal` and are a `GemmConfig` otherwise. Makes sense, concern withdrawn.